src/test/java/com/wikia/webdriver/common/remote/discussions/Categories.java

```diff
@@ -0,0 +1,55 @@
+package com.wikia.webdriver.common.remote.discussions;
+
+import com.jayway.jsonpath.*;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.remote.RemoteException;
+import com.wikia.webdriver.common.remote.discussions.context.CreateCategoryContext;
+import com.wikia.webdriver.common.remote.operations.http.GetRemoteOperation;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.category.CategoryPill;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Categories {
+
+  private class CategoriesNotFetched extends RuntimeException {
+    private CategoriesNotFetched(String message, RemoteException cause) {
+      super(message, cause);
+    }
+  }
+
+  private static final String GET_CATEGORIES_URL_SUFFIX = "%s/forums";
+  private final GetRemoteOperation remoteOperation;
+
+  Categories(User user) {
+    remoteOperation = new GetRemoteOperation(user);
+  }
+
+  public List<CategoryPill.Data> execute(final CreateCategoryContext context) {
+    try {
+      return getCategories(remoteOperation.execute(buildUrl(context)), context);
+    } catch(RemoteException e) {
+      PageObjectLogging.logError("error: ", e);
+      throw new CategoriesNotFetched("Could not fetch categories.", e);
+    }
+  }
+
+  private List<CategoryPill.Data> getCategories(String response, CreateCategoryContext ctxt) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getCategories" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)